hips/tiles/surveys.py

```diff
@@ -128,7 +128,13 @@ def base_url(self) -> str:
         """HiPS access url"""
         return self.data['moc_access_url'].rsplit('/', 1)[0]
 
+    @property
+    def tile_width(self) -> int:
+        """HiPS tile width"""
+        return int(self.data['hips_tile_width']) or 512
+
     def directory(self, ipix: int) -> int:
+        """Directory containing HiPS tile(s)"""
```

Comment:
This description isn't great. This returns an int, not a directory.
Can you be more specific?